apps/spruce/cypress/integration/myPatches/patchCard/dropdown_menu.ts

```diff
@@ -90,20 +82,6 @@ describe("Dropdown Menu of Patch Actions", () => {
     cy.dataCy("restart-version").should("be.disabled");
   });
 
-  it("'Add to commit queue' shows enqueue modal", () => {
-    getPatchCardByDescription(patchWithVersionOnCommitQueue).within(() => {
-      cy.dataCy("patch-card-dropdown").click();
-    });
-    cy.dataCy("enqueue-patch").should("exist");
```

Comment:
Should this be removed from the actual PatchCard component? If not, can this test be added back in somewhere where it belongs?

Reply:
it'll be deleted shortly and has no functionality since the commit queue is not usable at all but I can add it back